control-plane/src/egressproxy.rs

```diff
@@ -45,8 +43,18 @@ impl EgressProxy {
         Ok(())
     }
 
+    async fn connect<T: AsyncReadExt + AsyncWriteExt + Unpin>(
+        stream: T,
+        egress_domains: EgressDomains,
+    ) {
+        if let Err(e) = Self::handle_connection(stream, egress_domains).await {
+            eprintln!("An error occurred while handling an egress connection - {e:?}");
+        }
+    }
```

Comment:
This doesn't seem to add anything over using handle_connection directly?

Reply:
Oh yeah that was me trying to get away from move errors from the domains, I cleaned that up there